keylime/cloud_verifier_tornado.py

```diff
@@ -1596,7 +1598,10 @@ def sig_handler(*_: Any) -> None:
     num_workers = config.getint("verifier", "num_workers")
     if num_workers <= 0:
         num_workers = tornado.process.cpu_count()
+
+    agents = get_agents_by_verifier_id(verifier_id)
     for task_id in range(0, num_workers):
-        process = Process(target=server_process, args=(task_id,))
+        active_agents = [agents[i] for i in range(task_id, len(agents), num_workers)]
```

Comment:
I like this one. In 3.12 maybe we can use https://docs.python.org/3.12/library/itertools.html#itertools.batched

Reply:
I think what's there right now is better for equal distribution. I wouldn't want to neglect the last process started with a really short list versus the other ones having equally filled lists. :-) 

Until we all have 3.12 it's going to be a while...

Thanks.